BUILD.boost

```diff
@@ -125,6 +139,7 @@ BOOST_CTX_ASM_SOURCES = select({
         "libs/context/src/asm/jump_x86_64_ms_pe_masm.S",
         "libs/context/src/asm/ontop_x86_64_ms_pe_masm.S",
     ],
+    "//conditions:default": [],
```

Comment:
Do you believe that there are platforms for which this library will work without any assembly files? If not, I'd prefer to leave the `default` out, since I think the error message indicating that we haven't implemented that platform in bazel will be clearer than an obscure link failure. Same question for the ones below.

Reply:
The context library has asm for arm, arm64, i386, x86_64, mips32, ppc32, ppc64 which covers almost every major architecture, so any project using context library with bazel in those respective platforms may get an error while building(I guess, static linking) if they aren't included in the BUILD.boost.

> since I think the error message indicating that we haven't implemented that platform in bazel will be clearer than an obscure link failure

But, error message isn't clear when the ```default``` is not implemented.  The error message is,
```Configurable attribute "defines" doesn't match this configuration (would a default condition help?).```

Someone who doesn't know bazel build has to fiddle around BUILD.boost to fix this, I think that's unfortunate as standalone boost install on these platform for their respective package managers are trouble free.

Reply:
Yeah, OK, that error message is a bit worse than I'd hope, especially the suggestion to add a `default`. I think this is probably fine, and will at least let other platforms use other libraries, at a minimum.